guides/common/modules/ref_examples-of-organization-management-in-project.adoc

```diff
@@ -3,6 +3,9 @@
 [id="examples-of-using-organizations-in-{project-context}_{context}"]
 = Examples of using organizations in {Project}
 
+[role="_abstract"]
+You can use organizations to structure your content and system administration based on your business needs, whether you run a small business, a large company with multiple business units, or manage systems for external customers.
```

Comment:
I suggest to reword system > host ~~and business > organization~~ (weird is a module about organizations)

```suggestion
You can use organizations to structure your content and host management based on your business needs, whether you run a small business, a large company with multiple business units, or manage hosts for external customers.
```